_pytest/fixtures.py

```diff
@@ -783,8 +783,8 @@ def execute(self, request):
             self.finish()
             assert not hasattr(self, "cached_result")
 
-        ihook = self._fixturemanager.session.ihook
-        return ihook.pytest_fixture_setup(fixturedef=self, request=request)
+        hook = self._fixturemanager.session.config.hook
```

Comment:
@nicoddemus @RonnyPfannschmidt I disagree - `session.gethookproxy(fspath)` is specifically what I'm changing in this PR, because `session.ihook` does precisely `session.gethookproxy(self.fspath)` (https://github.com/pytest-dev/pytest/blob/master/_pytest/main.py#L267-L270) and it doesn't work.